kuma/static/styles/components/payments/_tooltip.scss

```diff
@@ -0,0 +1,89 @@
+.tooltip {
```

Comment:
It seems like a good idea to move this to a file of its own.

I suggest that you give the file a more specific name, like email-tooltip, since this includes absolute positioning that makes it specific to this one field in this one form

Reply:
And maybe also add a comment on this .tooltip class explaining that despite its generic name, it has hardcoded positioning and is only intended for use for one specific tooltip.